llvm/test/Analysis/LazyValueAnalysis/lvi-icmp-ne-of-and.ll

```diff
@@ -0,0 +1,91 @@
+; RUN: opt < %s -passes="jump-threading,print<lazy-value-info>" -disable-output 2>&1 | FileCheck %s
```

Comment:
I prefer not to have `print<lazy-value-info>` tests. If you want to have better test coverage for this logic, I would recommend adding a `ConstantRange::makeMastNotEqualRange()` or so helper and then test it in ConstantRangeTest -- the big advantage is that you can add an exhaustive test and be certain that the logic is always correct.

Reply:
Good point! Updated and dropped previous test, thanks.